### PR TITLE
Add new demos

### PR DESCRIPTION
**Pull request by agsfer**
Detect Chemicals in Medical Text (BertForTokenClassification)
Detect Adverse Drug Events (BertForTokenClassification)
Detect Anatomical and Observation Entities in Chest Radiology Reports
Sentence Entity Resolver for billable ICD10-CM HCC codes
Sentence Entity Resolver for ICD10-CM codes (Generalized version)
Detect Entities in tweets
Extract Chunk Key Phrases

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Code improvements with no or little impact
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [CONTRIBUTING](http://nlp.johnsnowlabs.com/contribute.html) page.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by josejuanmartinez — CHANGES_REQUESTED** [review]
I have some comments for @murat-gunay about two titles, the rest has been commented. When we are ok, we will send it to @agsfer

File: docs/demos/public_english_language.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+layout: demopage
+title: Public english language
```
[the remark this answers is not in the document]
Comment (murat-gunay) [reply docs/demos/public_english_language.md:3]: @agsfer hi
is this "public English language" affecting something on the main web page.. i don't get this change

File: docs/_includes/demomenu.html [hunk 2]
```diff
@@ -17,6 +17,9 @@
       <li {% if _section.secheader %} {% for child in _section.secheader %} {% if child.activemenu == "analyze_spelling_grammar" %} class="active"  {% endif %} {% endfor %} {% endif %} >
         <a href="/analyze_spelling_grammar">Analyze Spelling &amp; Grammar</a>
       </li>
+      <li {% if _section.secheader %} {% for child in _section.secheader %} {% if child.activemenu == "public_english_language" %} class="active"  {% endif %} {% endfor %} {% endif %} >
+        <a href="/public_english_language">Public english language</a>
```
[the remark this answers is not in the document]
Comment (murat-gunay) [reply docs/_includes/demomenu.html:21]: @agsfer 
is this "public English language" affecting something on the main web page.. I don't get this change

File: docs/demos/recognize_biomedical_entities.md [hunk 3]
```diff
@@ -116,4 +116,18 @@ data:
           - text: Colab Netbook
             type: blue_btn
             url: https://githubtocolab.com/JohnSnowLabs/spark-nlp-workshop/blob/master/tutorials/Certification_Trainings/Healthcare/1.Clinical_Named_Entity_Recognition_Model.ipynb
+        - title: Detect Chemicals in Medical Text (Bert For Token Classification)
```
[the remark this answers is not in the document]
Comment (murat-gunay) [reply docs/demos/recognize_biomedical_entities.md:119]: @agsfer can you change the title removing `(Bert For Token Classification)`

File: docs/demos/recognize_biomedical_entities.md [hunk 4]
```diff
@@ -116,4 +116,18 @@ data:
           - text: Colab Netbook
             type: blue_btn
             url: https://githubtocolab.com/JohnSnowLabs/spark-nlp-workshop/blob/master/tutorials/Certification_Trainings/Healthcare/1.Clinical_Named_Entity_Recognition_Model.ipynb
+        - title: Detect Chemicals in Medical Text (Bert For Token Classification) 
+          id: detect_chemicals_medical_text_bertfortokenclassification
+          image: 
+              src: /assets/images/Detect_Chemicals_in_Medical_Text.svg
+          image2: 
+              src: /assets/images/Detect_Chemicals_in_Medical_Text_f.svg
+          excerpt: This demo shows how chemical compounds can be extracted from medical texts using Spark NLP model which trained with BertForTokenClassifier.
```
[the remark this answers is not in the document]
Comment (murat-gunay) [reply docs/demos/recognize_biomedical_entities.md:125]: @agsfer ^^

File: docs/demos/recognize_clinical_entities.md [hunk 5]
```diff
@@ -228,4 +231,32 @@ data:
           - text: Colab Netbook
             type: blue_btn
             url: https://colab.research.google.com/github/JohnSnowLabs/spark-nlp-workshop/blob/master/tutorials/streamlit_notebooks/healthcare/NER_BERT_TOKEN_CLASSIFIER.ipynb
+        - title: Detect Adverse Drug Events (Bert For Token Classification)
```
[the remark this answers is not in the document]
Comment (murat-gunay) [reply docs/demos/recognize_clinical_entities.md:234]: @agsfer ^^

File: docs/demos/recognize_clinical_entities.md [hunk 6]
```diff
@@ -228,4 +231,32 @@ data:
...
+          id: detect_adverse_drug_events_bertfortokenclassification
+          image: 
+              src: /assets/images/Detect_Adverse_Drug_Events.svg
+          image2: 
+              src: /assets/images/Detect_Adverse_Drug_Events_f.svg
+          excerpt: This demo shows how adverse reactions of drugs in medical text can be detected using Spark NLP model which trained with BertForTokenClassifier.
```
[the remark this answers is not in the document]
Comment (murat-gunay) [reply docs/demos/recognize_clinical_entities.md:240]: @agsfer ^^

File: docs/demos/resolve_entities_codes.md [hunk 7]
```diff
@@ -19,7 +19,6 @@ data:
       source: yes
       source: 
         - title: SNOMED coding
```
[the remark this answers is not in the document]
Comment (murat-gunay) [reply docs/demos/resolve_entities_codes.md:21]: @agsfer ^^

File: docs/_includes/demomenu.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (josejuanmartinez) [reply docs/_includes/demomenu.html:21]: Hi @agsfer  @murat-gunay . Let's remove this section, that only has 1 demo. That demo (english tweet sentiment) should go to `Spark NLP: English - Infer meaning & Intent`

File: docs/demos/public_english_language.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (josejuanmartinez) [reply docs/demos/public_english_language.md:3]: Hi @agsfer  @murat-gunay . Let's remove this section, that only has 1 demo. That demo (english tweet sentiment) should go to `Spark NLP: English - Infer meaning & Intent`

File: docs/demos/infer_meaning_intent.md [hunk 8]
```diff
@@ -116,5 +116,18 @@ data:
...
             type: blue_btn
             url:  https://github.com/JohnSnowLabs/spark-nlp-workshop/blob/master/tutorials/streamlit_notebooks/NER.ipynb
-              
+        - title: Detect Entities in tweets 
+          id: detect_entities_tweets  
+          image: 
+              src: /assets/images/Detect_Entities_in_tweets.svg
+          image2: 
+              src: /assets/images/Detect_Entities_in_tweets_f.svg
+          excerpt: This demo shows how Broad Twitter Corpus (BTC) entities can be detected automatically using Spark NLP models.
```
Comment (josejuanmartinez) [inline docs/demos/infer_meaning_intent.md:125]: This demo shows how to extract Named Entities, as PER, ORG or LOC, from tweets.

File: docs/demos/recognize_clinical_entities.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (josejuanmartinez) [reply docs/demos/recognize_clinical_entities.md:240]: @agsfer This is still pending to fix

File: docs/demos/recognize_clinical_entities.md [hunk 9]
```diff
@@ -228,4 +231,32 @@ data:
...
+          - text: Colab Netbook
+            type: blue_btn
+            url: https://colab.research.google.com/github/JohnSnowLabs/spark-nlp-workshop/blob/master/tutorials/streamlit_notebooks/healthcare/NER_BERT_TOKEN_CLASSIFIER.ipynb
+        - title: Detect Anatomical and Observation Entities in Chest Radiology Reports
+          id: detect_anatomical_observation_entities_chest_radiology_reports 
+          image: 
+              src: /assets/images/Detect_Anatomical_and_Observation_Entities_in_Chest_Radiology_Reports.svg
+          image2: 
+              src: /assets/images/Detect_Anatomical_and_Observation_Entities_in_Chest_Radiology_Reports_f.svg
+          excerpt: This demo shows how Anatomical and Observation entities can be extracted from Chest Radiology Reports using Spark NLP model.
```
Comment (josejuanmartinez) [inline docs/demos/recognize_clinical_entities.md:254]: remove `using Spark NLP model`

File: docs/demos/resolve_entities_codes.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (josejuanmartinez) [reply docs/demos/resolve_entities_codes.md:21]: @agsfer pending fix

File: docs/demos/resolve_entities_codes.md [hunk 10]
```diff
@@ -34,7 +33,6 @@ data:
             type: blue_btn
             url: https://colab.research.google.com/github/JohnSnowLabs/spark-nlp-workshop/blob/master/tutorials/streamlit_notebooks/healthcare/ER_SNOMED.ipynb
         - title: ICDO coding
```
[the remark this answers is not in the document]
Comment (josejuanmartinez) [reply docs/demos/resolve_entities_codes.md:35]: @agsfer pending fix

File: docs/demos/resolve_entities_codes.md [hunk 11]
```diff
@@ -121,18 +118,46 @@ data:
...
             url: https://colab.research.google.com/github/JohnSnowLabs/spark-nlp-workshop/blob/master/tutorials/Certification_Trainings/Healthcare/11.1.Healthcare_Code_Mapping.ipynb#scrollTo=e5qYdIEv4JPL
-        - title: Resolve Clinical Entities to LOINC Codes
+        - title: LOINC Coding
           id: resolve_clinical_entities_codes_loinc
           image: 
               src: /assets/images/Resolve_Clinical_Entities_to_LOINC.svg
           image2: 
               src: /assets/images/Resolve_Clinical_Entities_to_LOINC_f.svg
-          excerpt: This demo shows how clinical entities can be automatically mapped to Logical Observation Identifiers Names and Codes (LOINC) using sentence resolvers.
+          excerpt: This demo shows how laboratory result-related entities can be automatically mapped to Logical Observation Identifiers Names and Codes (LOINC) using sentence resolvers.
```
[the remark this answers is not in the document]
Comment (josejuanmartinez) [reply docs/demos/resolve_entities_codes.md:127]: @agsfer pending fix

File: docs/demos/resolve_entities_codes.md [hunk 12]
```diff
@@ -121,18 +118,46 @@ data:
...
           actions:
           - text: Live Demo
             type: normal
             url: https://demo.johnsnowlabs.com/healthcare/ER_LOINC/
           - text: Colab Netbook
             type: blue_btn
-            url: https://githubtocolab.com/JohnSnowLabs/spark-nlp-workshop/blob/master/tutorials/Certification_Trainings/Healthcare/24.Improved_Entity_Resolvers_in_SparkNLP_with_sBert.ipynb             
+            url: https://githubtocolab.com/JohnSnowLabs/spark-nlp-workshop/blob/master/tutorials/Certification_Trainings/Healthcare/24.Improved_Entity_Resolvers_in_SparkNLP_with_sBert.ipynb    
+        - title: Sentence Entity Resolver for billable ICD10-CM HCC codes
```
[the remark this answers is not in the document]
Comment (josejuanmartinez) [reply docs/demos/resolve_entities_codes.md:135]: @agsfer pending fix

File: docs/demos/resolve_entities_codes.md [hunk 13]
```diff
@@ -121,18 +118,46 @@ data:
...
+          id: resolve_clinical_entities_codes_loinc
+          image: 
+              src: /assets/images/Sentence_Entity_Resolver_for_billable_ICD10-CM_HCC.svg
+          image2: 
+              src: /assets/images/Sentence_Entity_Resolver_for_billable_ICD10-CM_HCC_f.svg
+          excerpt: This demo shows how the extracted medical entities can be mapped to ICD-10 CM-HCC codes using Spark NLP for Healthcare sentence resolvers.
```
[the remark this answers is not in the document]
Comment (josejuanmartinez) [reply docs/demos/resolve_entities_codes.md:141]: @agsfer pending fix

File: docs/demos/resolve_entities_codes.md [hunk 14]
```diff
@@ -121,18 +118,46 @@ data:
...
+          actions:
+          - text: Live Demo
+            type: normal
+            url: https://demo.johnsnowlabs.com/healthcare/ER_ICD10_CM/
+          - text: Colab Netbook
+            type: blue_btn
+            url: https://colab.research.google.com/github/JohnSnowLabs/spark-nlp-workshop/blob/master/tutorials/streamlit_notebooks/healthcare/ER_ICD10_CM.ipynb  
+        - title: Sentence Entity Resolver for ICD10-CM codes (Generalized version)
```
[the remark this answers is not in the document]
Comment (josejuanmartinez) [reply docs/demos/resolve_entities_codes.md:149]: `Sentence Entity Resolver for ICD10-CM codes`

File: docs/demos/resolve_entities_codes.md [hunk 15]
```diff
@@ -121,18 +118,46 @@ data:
...
+          id: resolve_clinical_entities_codes_loinc
+          image: 
+              src: /assets/images/Sentence_Entity_Resolver_for_billable_ICD10-CM_HCC_codesReports.svg
+          image2: 
+              src: /assets/images/Sentence_Entity_Resolver_for_billable_ICD10-CM_HCC_codesReports_f.svg
+          excerpt: This demo shows how the extracted medical entities can be mapped to ICD-10 CM generalized codes using Spark NLP for Healthcare sentence resolvers.
```
[the remark this answers is not in the document]
Comment (josejuanmartinez) [reply docs/demos/resolve_entities_codes.md:155]: `This demo shows how medical terminology can be mapped to ICD-10 CM codes using Spark NLP for Healthcare. Entities have been grouped to provide a more high-level / generalized resolution`

[the remark this answers is not in the document]
Comment (josejuanmartinez) [reply docs/demos/resolve_entities_codes.md:155]: @agsfer

**Review by josejuanmartinez — CHANGES_REQUESTED** [review]
@agsfer  Can you take a look at the new changes required, and also, to the pending from previous review? I have tagged you in the pending ones.

Thank you!

File: docs/demos/analyze_non_english_medical_text.md [hunk 16]
```diff
@@ -19,7 +19,6 @@ data:
       source: yes
       source: 
         - title: ICD10 coding for German
```
Comment (josejuanmartinez) [inline docs/demos/analyze_non_english_medical_text.md:21]: ICD-10 taxonomy for German medical terminology

File: docs/demos/resolve_entities_codes.md [hunk 17]
```diff
@@ -18,8 +18,7 @@ data:
           activemenu: resolve_entities_codes
       source: yes
       source: 
-        - title: SNOMED coding
-          hide: yes
+        - title: SNOMED Coding
```
Comment (josejuanmartinez) [inline docs/demos/resolve_entities_codes.md:21]: Map clinical terminology to SNOMED taxonomy

File: docs/demos/resolve_entities_codes.md [hunk 18]
```diff
@@ -33,8 +32,7 @@ data:
           - text: Colab Netbook
             type: blue_btn
             url: https://colab.research.google.com/github/JohnSnowLabs/spark-nlp-workshop/blob/master/tutorials/streamlit_notebooks/healthcare/ER_SNOMED.ipynb
-        - title: ICDO coding
-          hide: yes
+        - title: ICD-O coding
```
Comment (josejuanmartinez) [inline docs/demos/resolve_entities_codes.md:35]: Map oncology terminology to ICD-O taxonomy

**Review by josejuanmartinez — CHANGES_REQUESTED** [review]
Last 3 small changes and we are ready to go :) Thank you!

**Review by josejuanmartinez — APPROVED** [review]
Reviewed and approved. More changes may be required later on, but at least this can be merged.

**Comment by vkocaman** [comment]
good job everyone !

**Pull request merged**